camel/configs/anthropic_config.py

```diff
@@ -15,7 +15,7 @@
 
 from typing import List, Union
 
-from anthropic import NOT_GIVEN, NotGiven
+from openai._types import NOT_GIVEN, NotGiven
```

Comment:
Import `openai` for `anthropic` types seems weird. Can we import `anthropic` inside the class instead?